Add "Hibernate" option, add a possible workaround for nosystemd systems

ItsZariep opened the pull request:
# Add "Hibernate" Option
 - Useful for those who want to hibernate their system with the icewm menu itself
 - Added option in prefs, wmdialog and menu

# Add a workaround for nosystemd systems
- Simply added ```|| loginctl ... ``` at the end of all session commands
 
 # Summary
 Changes to be committed:
	modified:   man/icewm-preferences.pod
	modified:   src/default.h
	modified:   src/icesh.cc
	modified:   src/wmaction.h
	modified:   src/wmapp.cc
	modified:   src/wmdialog.cc
	modified:   src/wmdialog.h
	modified:   src/wmmgr.cc

Commit:
Add "Hibernate" option, add a possible workaround for nosystemd systems
 Changes to be committed:
	modified:   man/icewm-preferences.pod
	modified:   src/default.h
	modified:   src/icesh.cc
	modified:   src/wmaction.h
	modified:   src/wmapp.cc
	modified:   src/wmdialog.cc
	modified:   src/wmdialog.h
	modified:   src/wmmgr.cc

gijsbers commented:
1. What is the difference between suspend and hibernate?
2. You add a trailing space in icesh.cc.
3. The numerical values for WMActions can't be changed.
4. The manpage for my loginctl (systemd) doesn't mention suspend/hibernate/poweroff.
5. What operating system is your loginctl from?

ItsZariep commented:
1.- Hibernate mode is a deeper power-saving state where the computer saves the system state to disk and shuts down completely.
2.- sorry for that, maybe i pressed Space unintentionally while editing the files
3.- i understand that, maybe replacing some option since exit and cancel do basically the same thing
4.- Arch manpages doesn't include it, maybe a difference between systemd and elogind, Void manpage include it
5.- i tried Artix and Void

gijsbers commented:
Can you add a commit which removes the trailing space from icesh.cc and which keeps existing numerical values for WMAction intact? Then I can squash and merge and nothing will remain visible.

Commit:
removes the trailing space from icesh.cc and undo wmaction

ItsZariep commented:
done, i removed    ` actionHibernate          = 174,`  from icesh.cc

gijsbers commented:
No, you still have the modified numerical values for WMActions, which breaks compatibility.
`ICEWM_ACTION_HIBERNATE` should be appended as the highest value.
Idem for `actionHibernate` which should be an odd number two higher than the currently highest.

Commit:
do requested: ICEWM_ACTION_HIBERNATE should be appended as the highest value.
Idem for actionHibernate which should be an odd number two higher than the currently highest.
